app/views/handlers/block.py

```diff
@@ -29,6 +29,10 @@ def __init__(self, schema, questionnaire_store, language, current_location):
                 f'location {self._current_location} is not valid'
             )
 
+    @property
```

Comment:
Can this just expose current_location rather than making this a property?

Reply:
Can do, only did this to prevent it from being set?

Reply:
Yep, fair enough